x-pack/test/security_solution_endpoint_api_int/apis/file_upload_index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esClient = getService('es');
+
+  describe('File upload indices', () => {
+    it('should have created the file data index on install', async () => {
+      const endpointFileUploadIndexExists = await esClient.indices.exists({
+        index: '.fleet-file-data-endpoint',
```

Comment:
can we use the `const`'s we define here for the index name here and below?

https://github.com/elastic/kibana/blob/6b6cdf8ab7eedd1d6a93bd0815ae8f416f47c239/x-pack/plugins/security_solution/common/endpoint/constants.ts#L47-L48